sys/freebsd/dev_smbus.txt.const

```diff
@@ -1,5 +1,5 @@
 # Code generated by syz-sysgen. DO NOT EDIT.
-arches = 386, amd64
+arches = 386, amd64, riscv64
```

Comment:
Did you actually regenerate the const files? Or changed manually?
E.g. here there is no value for riscv64:
```
SMB_BREAD = 386:3223087371, amd64:3224398091
```

Reply:
Fixed in https://github.com/google/syzkaller/pull/3253/commits/ccdde1002bb2771dd1e2c638552c8a011a202643 & https://github.com/google/syzkaller/pull/3253/commits/f8c8d36975c7bd1365abb11dab4dfe9d44447015 .